jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalecen" -c "base ocnanal ocnanalecen"
```

Comment:
why is `ocnanal` config needed in this task?

Reply:
For `JEDI_BIN`

Reply:
What is in `JEDI_BIN`? All executables are in `HOMEgfs/exec`

Reply:
Also for `SOCA_INPUT_FIX_DIR`